plugins/available/base.plugin.bash

```diff
@@ -4,13 +4,24 @@ about-plugin 'miscellaneous tools'
 function ips ()
 {
     about 'display all ip addresses for this host'
+    param 'optional -i'
+    example '$ ips'
+    example '$ ips -i'
     group 'base'
     if command -v ifconfig &>/dev/null
     then
-        ifconfig | awk '/inet /{ gsub(/addr:/, ""); print $2 }'
+        if [ -n "$1" -a "$1" = "-i" ];then
+            ifconfig | grep -B 1 -E "inet "| sed 's/addr://g' | sed 's/: / /g' | awk '{if (NR%3==1) print $1 ":"; else if (NR%3==2) print $2}'
```

Comment:
Do you really have to run grep, sed (twice) and awk all in the same line? I'm sure there's got to be a way that does not use three different tools to accomplish this.